### PR TITLE
Add status page and update warning.

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Here's how it looks:
```r
y <- readNWISqw("391259097001800", parameterCd = "00605")
Warning message:                                                                                     
WARNING: Beginning in February 2024 the NWIS qw data endpoint will not deliver new data or updates to existing data. 
Eventually the endpoint will be retired. For updated information visit:
https://waterdata.usgs.gov/nwis/qwdata
For additional details, see vignettes:
https://doi-usgs.github.io/dataRetrieval/articles/Status.html
https://doi-usgs.github.io/dataRetrieval/articles/qwdata_changes.html
If you have additional questions about the qw data service, email CompTools@usgs.gov
```

We can merge this PR and make sure the documentation deploys as expected. Here's how the banner looks on my screen locally:

image

### CODE REVIEW

File: vignettes/Status.Rmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+See [Changes to NWIS QW services](articles/qwdata_changes.html) for information on how to convert your workflows from `readNWISqw` to `readWQPqw`.
+
+## WQP: USGS Data
+
+Limited availability of new USGS data: starting approximately February 12, 2024, there will be a period when new USGS data will not be accessible on the Water Quality Portal. Data will still be collected but will not be publicly available. This limited availability is expected to last a few weeks.
+
+Recent USGS data will be accessible again upon release of the new Data Profiles, which will be in the WQX 3.0 format. These profiles should be available in March, 2024.
+
+These profiles are not yet available to test. When the are, we will update this page with recommendations.
```
Comment (lstanish-usgs) [inline vignettes/Status.Rmd:56]: misspelling: the --> they

File: vignettes/Status.Rmd [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+It is planned that when NWIS qw service will be shut down - the `readNWISqw` will be removed from dataRetrieval.
+
+
+See [Changes to NWIS QW services](articles/qwdata_changes.html) for information on how to convert your workflows from `readNWISqw` to `readWQPqw`.
+
+## WQP: USGS Data
+
+Limited availability of new USGS data: starting approximately February 12, 2024, there will be a period when new USGS data will not be accessible on the Water Quality Portal. Data will still be collected but will not be publicly available. This limited availability is expected to last a few weeks.
+
+Recent USGS data will be accessible again upon release of the new Data Profiles, which will be in the WQX 3.0 format. These profiles should be available in March, 2024.
```
Comment (lstanish-usgs) [inline vignettes/Status.Rmd:54]: update text: 'should be available starting in March'

File: vignettes/Status.Rmd [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+If you have additional questions about the qw data service, email CompTools@usgs.gov.
+
+It is planned that when NWIS qw service will be shut down - the `readNWISqw` will be removed from dataRetrieval.
+
+
+See [Changes to NWIS QW services](articles/qwdata_changes.html) for information on how to convert your workflows from `readNWISqw` to `readWQPqw`.
+
+## WQP: USGS Data
+
+Limited availability of new USGS data: starting approximately February 12, 2024, there will be a period when new USGS data will not be accessible on the Water Quality Portal. Data will still be collected but will not be publicly available. This limited availability is expected to last a few weeks.
```
Comment (lstanish-usgs) [inline vignettes/Status.Rmd:52]: can you emphasize the 'new' data part? could be easy for people to read and think this applies to all USGS data.

File: vignettes/Status.Rmd [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## WQP
+
+Functioning normally, includes new USGS data. See below for upcoming information.
+
+# Overview of changes
+
+## NWIS qw services
+
+Discrete water samples data are undergoing modernization, and NWIS services will no longer being updated with the latest data starting in February 2024, with a full decommission expected 6 months later. Learn more about the upcoming change and where to find the new samples data in our [blog](https://waterdata.usgs.gov/blog/changes-to-sample-data/).
```
Comment (lstanish-usgs) [inline vignettes/Status.Rmd:34]: misspelling: will --> with

**Comment by lstanish-usgs** [comment]
@ldecicco-USGS Updates look great! I like the new dedicated 'Status' page as well. That will make it easier to update folks about changes. I noted a few typos and minor edits to the Status page. Pending those fixes merging is approved! Thanks

File: vignettes/Status.Rmd (hunk 4 above)
Comment (ldecicco-USGS) [reply vignettes/Status.Rmd:34]: sorry, what "will" needs to change? I'm not seeing it

Comment (lstanish-usgs) [reply vignettes/Status.Rmd:34]: sorry that wasn't clear: on reading again, this makes more sense: 'and NWIS services will no longer be updated...', so change being-->be instead.

**Pull request merged**